src/components/select-button.tsx

```diff
@@ -40,7 +40,7 @@ export const SelectButton: React.FC<ISelectButtonProps> = (props) => {
           onClose={onMenuClose}
         >
           {menuItems.map(item => (
-            <MenuItem key={item.value} className={css.menuItem} value={item.value}>
+            <MenuItem key={item.value} value={item.value}>
```

Comment:
I removed this class because it didn't seem possible to use it for styling the menu item elements. I instead modified `src/material-ui-theme.tsx` for a required font size change.